tests/tests/test_github_webhooks.py

```diff
@@ -436,6 +442,7 @@ def test_issue_comment_minor_series(integration_test_runner_url):
         "info:mender:865 will trigger 1 builds",
         'info:1: (main.buildOptions) {\n pr: (string) (len=3) "865",\n repo: (string) (len=6) "mender",\n baseBranch: (string) (len=5) "3.1.x",\n commitSHA: (string) (len=40) "75ad5f739a6e0bd3367e92d846521a85a4e8bb35",\n makeQEMU: (bool) true\n}\n\n',
         "info:auditlogs version 2.0.x is being used in 3.1.x",
+        "info:azure-iot-manager version 4.1.x is being used in 3.1.x",
```

Comment:
4.1.x? What?

Reply:
correction: it has to be empty for now. @lluiscampos would you consider this one acceptable?correction: it has to be empty for now. @lluiscampos would you consider this one acceptable?

Reply:
gracias amigo